sensor_msgs/include/sensor_msgs/image_encodings.hpp

```diff
@@ -186,7 +188,9 @@ static inline int numChannels(const std::string & encoding)
     }
   }
 
-  if (encoding == YUV422) {
+  if (encoding == YUV422 ||
```

Comment:
Can you please add tests to https://github.com/ros2/common_interfaces/blob/81663c07b93889c3d0afda9b99cd5f1c7c98c1f2/sensor_msgs/test/test_image_encodings.cpp for `numChannels` and `bitDepth`?  It looks like this was missed when the YUV422 one was added as well, so we may as well add tests for both that one and the new one.  Thanks!

Reply:
Sure thing, done!

Reply:
Thanks!  I'll run CI on this now.

Reply:
Great thanks! I guess the failures are not related to this?

Reply:
The Linux failures were an infrastructure issue; the job got automatically restarted, but it doesn't propagate that back to the issue, so I just manually updated it.  It looks good now.

The Windows failure was because of a typo in how I launched it.  I've now fixed that typo and its running again.